app/common/services/maps.js

```diff
@@ -58,11 +59,34 @@ function Maps(ConfigEndpoint, L, _, CONST) {
 
             // Add a layer control
             // L.control.layers(getBaseLayersForControl(), {}).addTo(map);
+            var resetButton  = L.easyButton({
+                id: 'reset-button',
+                position: 'bottomleft',
+                type: 'replace',
+                leafletClasses: true,
+                states:[{
+                    // specify different icons and responses for your button
+                    stateName: 'reset-button',
+                    onClick: function() {
+                        var defaultview = defaultValues();
+                        map.setView([defaultview.lat, defaultview.lon], defaultview.zoom);
+                    },
+                    title: 'Reset to default view',
+                    icon: '<i class = "fa fa-home"></i>'
+                }]
+            });
+
+            resetButton.addTo(map);
 
             return map;
         });
     }
 
+    function setDefaultView() {
```

Comment:
Is this function used? I cannot find it but maybe its a Leaflet-default-thing?

Reply:
Yes, you're right :). I should have seen that earlier. This piece of code is redundant.

> Is this function used? I cannot find it but maybe its a Leaflet-default-thing?

Yes, you are right :). This piece of code is redundant. I should have seen that earlier.

Reply:
> Is this function used? I cannot find it but maybe its a Leaflet-default-thing?

Yes, you are right. This piece of code is redundant. I should have seen that earlier :)